code/__HELPERS/atmospherics.dm

```diff
@@ -6,3 +6,30 @@
 
 /proc/molar_cmp_equals(a,b,epsilon = MAXIMUM_ERROR_GAS_REMOVAL)
 	return (((a + epsilon) > b) && ((a - epsilon) < b))
+
+/** A simple rudimentary gasmix to information list converter. Can be used for UIs.
+ * Args: 
+ * - gasmix: [/datum/gas_mixture]
+ * - name: String used to name the list, optional.
+ * Returns: A list parsed_gasmixes with the following structure:
+ * - parsed_gasmixes - Assoc List
+ * -- Key: name			Value: String			Desc: Gasmix Name
+ * -- Key: temperature		Value: Number			Desc: Temperature in kelvins
+ * -- Key: volume 			Value: Number			Desc: Volume in liters
+ * -- Key: pressure 		Value: Number			Desc: Pressure in kPa
+ * -- Key: ref				Value: Text				Desc: The reference for the instantiated gasmix.
+ * -- Key: gases			Value: Assoc list		Desc: List of gasses in our gasmix
+ * --- Key: gas_name 		Value: Gas Mole			Desc: Gas Name - Gas Amount pair
+ * Returned list should always be filled with keys even if value are nulls.
+ */
+/proc/gas_mixture_parser(datum/gas_mixture/gasmix, name = null)
```

Comment:
you don't need to set null explicitly like this, if no argument is provided we default to it anyway